index.html

```diff
@@ -53,7 +53,23 @@
         </section>
         <!-- end banner announcement -->
 
+        
+        <!-- begin standard header to be hidden when event like community meeting is announced -->
+        <header class="marketing-hero" id="bg-image-main">
+            <div class="row homepage text-center">
```

Comment:
You need to add class "bg-image-text-container" here.
Also the -begin banner announcement- section above needs to be inside the commented-out section below.
It was added for the OME meeting with "register now". Then the "register now" was removed when the registration closed but I didn't remove the banner itself since the page would look wrong without the red line.